### PR TITLE
ENT-10414: Made lastseenexpireafter in body common control configurable via Augments

### PR DESCRIPTION
**Pull request by nickanderson**

### CODE REVIEW

File: controls/def.cf [hunk 1]
```diff
@@ -246,6 +246,10 @@ bundle common def
                           "true",
                           "false");
 
+      "control_common_lastseenexpireafter" -> { "ENT-10414" }
+        string => "10080", # 10080 minutes is 1 week
```
Comment (craigcomstock) [inline controls/def.cf:250]: Be nice to mention that this is also the default in the binaries right?

https://docs.cfengine.com/docs/3.21/reference-components.html#lastseenexpireafter

File: MPF.md [hunk 2]
```diff
@@ -795,6 +795,27 @@ This example illustrates enabling the option via augments.
...
+### lastseenexpireafter
+
+This option configures the number of minutes after which last-seen entries in `cf_lastseen.lmdb` are purged.
+
+
+```json
+{
+  "variables": {
+    "default:def.control_common_ignore_missing_inputs": {
+      "value": "30240",
```
Comment (craigcomstock) [inline MPF.md:807]: So the specified default later

```
        string => "10080", # 10080 minutes is 1 week
```

Is also the default in the binaries I guess: https://docs.cfengine.com/docs/3.21/reference-components.html#lastseenexpireafter

File: controls/def.cf (hunk 1 above)
Comment (nickanderson) [reply controls/def.cf:250]: Sure, can do.

File: MPF.md (hunk 2 above)
Comment (nickanderson) [reply MPF.md:807]: Indeed, we mirror binary defaults in the MPF as a general rule but call out when we deviate from the binary defaults.

File: controls/def.cf (hunk 1 above)
Comment (nickanderson) [reply controls/def.cf:250]: Done.

File: MPF.md (hunk 2 above)
Comment (nickanderson) [reply MPF.md:807]: Noted.

**Review by craigcomstock — APPROVED** [review]
awesome :)

**Comment by nickanderson** [comment]
@cf-bottom jenkins please

**Comment by cf-bottom** [comment]
Alright, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/9627/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/9627/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-9627/

**Pull request merged**